### PR TITLE
Fix peerdependencies for file-loader & react-tween-state

### PR DESCRIPTION
**Pull request by daphnesmit**
`npm ls` issue #258

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +54,7 @@
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-hot-loader": "^1.3.0",
-    "sinon": "^1.17.1",
+    "sinon": "^2.1.0",
```
Comment (nickspiel) [inline package.json]: Does sinon need to be updated?

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "eslint-loader": "^1.0.0",
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-react": "^7.4.0",
+    "file-loader": "^0.11.2",
```
Comment (nickspiel) [inline package.json]: Does file loader need to be updated as part of this PR?

Comment (daphnesmit) [reply package.json]: It  is added because it was missing

File: package.json (hunk 1 above)
Comment (daphnesmit) [reply package.json]: Probably not.  No idea how that happened lol! I also updated all packages for  webpack 3 etc in https://github.com/FormidableLabs/nuka-carousel/pull/261

**Comment by ajoshi0** [comment]
@kenwheeler can we get this merged need to use this for react-16 upgrade

**Comment by ajoshi0** [comment]
or @nickspiel ^^

**Comment by ryan-roemer** [comment]
Superseded by https://github.com/FormidableLabs/nuka-carousel/pull/275

**Pull request closed without merging**